Multiple flaky GUI tests due to `QTextEdit has been deleted` in GUI

```
=================================== FAILURES ===================================
_________ test_that_stdout_and_stderr_buttons_react_to_file_content[0] _________
CALL ERROR: Exceptions caught in Qt event loop:
________________________________________________________________________________
Traceback (most recent call last):
  File "/tmp/f_scout_ci/actions-runner-01/_temp/test_root/tests/ert/unit_tests/gui/simulation/test_run_dialog.py", line 594, in <lambda>
    QTimer.singleShot(100, lambda: handle_error_dialog(run_dialog))
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tmp/f_scout_ci/actions-runner-01/_temp/test_root/tests/ert/unit_tests/gui/simulation/test_run_dialog.py", line 585, in handle_error_dialog
    text = error_dialog.details_text.toPlainText()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: wrapped C/C++ object of type QTextEdit has been deleted
```

https://github.com/equinor/komodo-releases/actions/runs/13599327245/job/38023223009
https://github.com/equinor/komodo-releases/actions/runs/13610599730/job/38047591820
https://github.com/equinor/ert/actions/runs/13940885832/job/39017355122

--- a/src/ert/config/forward_model_step.py
+++ b/src/ert/config/forward_model_step.py
@@ -185,7 +185,6 @@ class ForwardModelStep:
         Raises ConfigValidationError if not all required keywords are in
         private_args
         """
-        print(self.required_keywords)
         missing_keywords = set(self.required_keywords).difference(self.private_args)
         if missing_keywords:
             plural = "s" if len(missing_keywords) > 1 else ""

--- a/tests/ert/unit_tests/gui/simulation/test_run_dialog.py
+++ b/tests/ert/unit_tests/gui/simulation/test_run_dialog.py
@@ -564,12 +564,10 @@ def test_that_exception_in_base_run_model_is_handled(qtbot: QtBot, storage):
     args_mock.config = config_file
 
     ert_config = ErtConfig.from_file(config_file)
-    with (
-        patch.object(
-            ert.run_models.SingleTestRun,
-            "run_experiment",
-            MagicMock(side_effect=ValueError("I failed :(")),
-        ),
+    with patch.object(
+        ert.run_models.SingleTestRun,
+        "run_experiment",
+        MagicMock(side_effect=ValueError("I failed :(")),
     ):
         gui = _setup_main_window(ert_config, args_mock, GUILogHandler(), storage)
         qtbot.addWidget(gui)

--- a/tests/ert/unit_tests/gui/simulation/test_run_dialog.py
+++ b/tests/ert/unit_tests/gui/simulation/test_run_dialog.py
@@ -575,7 +575,10 @@ def test_that_exception_in_base_run_model_is_handled(qtbot: QtBot, storage):
         qtbot.addWidget(gui)
         run_experiment = gui.findChild(QToolButton, name="run_experiment")
 
+        handler_done = False
+
         def handle_error_dialog(run_dialog):
+            nonlocal handler_done
             qtbot.waitUntil(
                 lambda: run_dialog.fail_msg_box is not None,
                 timeout=20000,
@@ -585,6 +588,7 @@ def test_that_exception_in_base_run_model_is_handled(qtbot: QtBot, storage):
             text = error_dialog.details_text.toPlainText()
             assert "I failed :(" in text
             qtbot.mouseClick(error_dialog.box.buttons()[0], Qt.MouseButton.LeftButton)
+            handler_done = True
 
         simulation_mode_combo = gui.findChild(QComboBox)
         simulation_mode_combo.setCurrentText("Single realization test-run")
@@ -593,6 +597,7 @@ def test_that_exception_in_base_run_model_is_handled(qtbot: QtBot, storage):
 
         QTimer.singleShot(100, lambda: handle_error_dialog(run_dialog))
         qtbot.waitUntil(lambda: run_dialog.is_simulation_done() == True, timeout=100000)
+        qtbot.waitUntil(lambda: handler_done, timeout=100000)
 
 
 @pytest.mark.integration_test

